### PR TITLE
Release/v2.7.2

### PR DESCRIPTION
**Pull request by ejether**

### CODE REVIEW

**Comment by ejether** [comment]
@robscott you made the initial audit policy changes to pentagon I think. Would you confirm that they are all present and added after running this migration?

**Review by robscott — APPROVED** [review]
Looks good to me, thanks!

File: pentagon/migration/__init__.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ def _write_new_version(self, version):
 
     def _append_migration_readme(self):
         if hasattr(self, "_readme_string"):
-            with open(migration_readme_file, 'w+') as mrf:
+            with open(migration_readme_file, 'a+') as mrf:
```
Comment (endzyme) [inline pentagon/migration/__init__.py:196]: `mrf` https://www.youtube.com/watch?v=HVIdy9_wqQw :D

File: pentagon/migration/migrations/migration_2_7_1.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
+from copy import deepcopy
+
+
+from pentagon import migration
+from pentagon.migration import *
```
Comment (endzyme) [inline pentagon/migration/migrations/migration_2_7_1.py]: Are both these imports needed or could one be summarized? Feels like we're importing the same set of information twice just with different scope contexts.

File: pentagon/migration/migrations/migration_2_7_1.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+            for cluster_item in os.listdir('{}/clusters/'.format(inventory_path)):
+                item_path = '{}/clusters/{}'.format(inventory_path, cluster_item)
+                # Is this a kops cluster?
+
+                # There is a small amount of variation here where some cluster config
+                # directories are `cluster` and some are `cluster-config`
+                # Align these
+                if os.path.isdir("{}/cluster".format(item_path)):
+                    self.move("{}/cluster".format(item_path), "{}/cluster-config".format(item_path))
```
Comment (endzyme) [inline pentagon/migration/migrations/migration_2_7_1.py:81]: Can we add output that we're doing this and why (basically the comment in a logging output to the person running the migration).

File: pentagon/migration/migrations/migration_2_7_1.py [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+class Migration(migration.Migration):
+    _starting_version = '2.7.1'
+    _ending_version = '2.7.2'
+
+    _readme_string = readme
+
+    def run(self):
+
+        old_nodes_file_name = "nodes.yml"
```
Comment (endzyme) [inline pentagon/migration/migrations/migration_2_7_1.py]: appears to be defined but not used? Is this used anywhere?

**Review by endzyme — APPROVED** [review]
Minimal optional changes - I tested this on a repo and it didn't fail - :+1:

File: pentagon/migration/migrations/migration_2_7_1.py (hunk 2 above)
Comment (ejether) [reply pentagon/migration/migrations/migration_2_7_1.py]: Good point. If there was a reason, I don't recall what it is was. It should be usable with just the import on line 4

File: pentagon/migration/migrations/migration_2_7_1.py (hunk 4 above)
Comment (ejether) [reply pentagon/migration/migrations/migration_2_7_1.py]: Nope, copy pasta leftovers

**Pull request merged**